src/core/lib/iomgr/buffer_list.cc

```diff
@@ -30,6 +32,10 @@
 #include <string.h>
 #include <time.h>
 
+GPR_GLOBAL_CONFIG_DEFINE_INT32(
```

Comment:
@soheilhy any experience on why folks would change this?

Reply:
I think it's fine to hard-code.

I believe this CL was designed after cl/331007927, then the default value (10s) has been working fine and I'm not aware of any case that overrides. At that time this value was chosen arbitrarily and I needed a control knob.

Reply:
Removed the flag now and hard-coded it.